### PR TITLE
 add new shells components

### PR DESCRIPTION
**Pull request by deepakgithub11**
This PR adds New Shells for WPUI Components

### CODE REVIEW

**Comment by ajitbohra** [comment]
one pattern from Figma design is missing also we need to test them for responsive and get some design feedback!

**Comment by ajitbohra** [comment]
Also we can use this branch to introduce new category for shells!

File: src/categories/index.js [hunk 1]
```diff
@@ -11,3 +11,4 @@ export { default as PageHeading } from './page-heading';
 export { default as SignIn } from './sign-in';
 export { default as Stats } from './stats';
 export { default as Tables } from './tables';
+export { default as shells } from './shells';
```
Comment (Verma-Punit) [inline src/categories/index.js:14]: **s** should be capital

**Comment by deepakgithub11** [comment]
Ready for Design Review

**Comment by deepakgithub11** [comment]
Ready for code view
Note: Changes have been applied only to Shell No. 2 for now. 
Once the design is approved, all shells will be updated to reflect both designs, and the code will be refactored accordingly.

**Pull request closed without merging**